tox.ini

```diff
@@ -1,4 +1,4 @@
 [flake8]
-ignore = F401,E501,E402,E401
+ignore = F401,E501,E402,E401, E305, E302, N812
```

Comment:
Also, while you're there, might as well delete the next line, since it's commented out anyway.